src/sentry/options/defaults.py

```diff
@@ -2563,6 +2563,18 @@
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+register(
+    "discover.saved-query-dataset-split.enable",
+    default=False,
+    flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
+)
+register(
+    "discover.saved-query-dataset-split.organization-allowlist",
+    type=Sequence,
+    default=[],
+    flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
+)
```

Comment:
Is there a benefit to us setting them as options instead of Flagpole? iirc Flagpole works for ST now

Reply:
The first one is a kill switch - just bypasses the whole task or forces us to break out of the task, this should just be an on/off not tied to anything so we can quickly shut it down in case it causes an incident

The second one, I kinda went back and forth on a little, my thinking was to initially just limit the queryset to the allowlisted orgs https://github.com/getsentry/sentry/pull/76201/files#diff-179f670f3f1385148441a5abb704eec14d23b90a6fbace8ca071996985c17815R54
instead of looping through the whole queryset and checking individual orgs. Then once we're done testing, just remove that option and run the task on dataset selector enabled orgs. I don't have strong opinions on this one though